runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -110,6 +110,9 @@
 	@Getter
 	private TrayIcon trayIcon;
 
+	@Getter
+	private JPanel container;
```

Comment:
Revert this. Just use Applet

Reply:
don't; this should work on things outside the applet

Reply:
Why? It should work only in game and not outside of it.

Reply:
it should work on our entire UI